security/faq.md

```diff
@@ -0,0 +1,71 @@
+---
+title: FAQ
+page_title: FAQ
+description: "Find answers to common questions about securing Telerik UI for Blazor components, including how to report vulnerabilities, handle third-party dependencies, and receive security fixes."
+slug: security-faq
+tags: telerik, blazor, security
+published: True
+position: 3
+---
+
+# Frequently Asked Questions (FAQ)
+
+In this article, you will find essential information and resources to help you secure Telerik UI for Blazor components. Whether you need to report vulnerabilities, manage third-party dependencies, or understand how security fixes are delivered, this FAQ provides clear guidance on our security processes. Explore best practices and learn how Progress ensures the highest level of security for its products, from vulnerability reporting to compliance with industry standards.
+
+## How can I submit a security vulnerability report?
+
+If you have identified a potential security vulnerability in a Telerik or Kendo UI product, you can report it through the following channels:
+
+- **For Progress Customers**: Submit a security report by opening a support ticket through the [Technical Support Center](https://www.telerik.com/account/support-center). Provide detailed information, including the steps to reproduce the issue and any relevant reports or screenshots.
+
+- **For Security Researchers**: Ethical hackers and security researchers can report vulnerabilities through our [Bugcrowd Vulnerability Disclosure Program](https://bugcrowd.com/engagements/whatsupgold-vdp). This platform ensures that security issues are addressed efficiently and transparently.
+
+We will review the report and follow up in line with our security processes, making every effort to resolve confirmed vulnerabilities in a timely manner.
+
+---
+
+## What if clients are using a third-party scanning tool and want our assessment?
+
+Clients can open a support ticket through the [Technical Support Center](https://www.telerik.com/account/support-center) for the respective product and submit detailed information, including scan reports (PDF/Word/Excel/Screenshot) triggered against a no-minified version of the scripts and steps to reproduce or evidence of the issue. Our team will review and respond accordingly. We treat Security Vulnerability reports with **highest priority** and we engage with our internal Security Champions for revisions.
```

Comment:
Who is "clients" ? Our clients or their clients? The latter can't submit tickets.

Reply:
```suggestion
As a client, you can open a support ticket through the [Technical Support Center](https://www.telerik.com/account/support-center) for the respective product and submit detailed information, including scan reports (PDF/Word/Excel/Screenshot) triggered against a no-minified version of the scripts and steps to reproduce or evidence of the issue. Our team will review and respond accordingly. We treat Security Vulnerability reports with **highest priority** and we engage with our internal Security Champions for revisions.
```